test/log-generators/cronjobs/install.sh

```diff
@@ -1,9 +1,48 @@
 #!/bin/bash
+# Copyright KubeArchive Authors
+# SPDX-License-Identifier: Apache-2.0
 
 # Create a CronJob that run every minute, creating an Apache log file 1024 lines long.
 
 SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
-namespace="generate-logs-cronjobs"
-kubectl create ns ${namespace}
-kubectl -n ${namespace} apply -f ${SCRIPT_DIR}
+# Parse command line arguments
+for i in "$@"
+do
+case $i in
+    --namespace=*)
+    NAMESPACE=`echo $i | sed 's/[-a-zA-Z0-9]*=//'`
+    ;;
+    --help)
+    HELP=True
+    ;;
+    *)
+    echo "Unknown option $i" # unknown option
+    HELP=True
+    UNKNOWN=True
+    ;;
+esac
+done
+
+HELP=${HELP:-""}
+UNKNOWN=${UNKNOWN:-""}
+export NAMESPACE=${NAMESPACE:-generate-logs-cronjobs}
+
+# Help and usage
+if [ "${HELP}" == "True" ] || [ "${UNKNOWN}" == "True" ]; then
+    echo -e "$0
+
+    --namespace    Namespace to use to log generation.
```

Comment:
Do you mean "...use to generate logs"?

```suggestion
    --namespace    Namespace to use to generate logs.
```

Reply:
I meant "for log generation", but your phrasing works too.

Fixed.